src/main/webapp/user_profile_style.css

```diff
@@ -125,28 +125,28 @@ th, td {
     padding: 15px;
```

Comment:
It won't let me add a comment for line 61, but this is meant for line 61: In the `#container .help-button` styling part, the buttons's text on my screen were wrapping around into 2 lines and getting near the bottom edge of the button, I think the button shrinks when the screen isn't as wide and there isn't enough space for the text. If you just delete the `width: 8%` part it seems like it doesn't do this anymore :)

Reply:
Oh I forgot to make the text responsive to the viewport size. I fix this problem in the newest commit.